pkg/controller/versions/strategy_v2_1.go

```diff
@@ -312,12 +311,8 @@ func (v *versionStrategyV2_1) Render(ctx context.Context, cr *common.ControllerR
 		return nil, fmt.Errorf("Unexpected error setting Status.AppliedSpec: %v", err)
 	}
 
-	// Read in global.yaml
-	values, err := chartutil.ReadValuesFile(path.Join(v.GetChartsDir(), "global.yaml"))
```

Comment:
@luksa if we want to bring back `global.yaml`, we will have to revert this code.